ingest-app/test/cmr/ingest/test/services/jobs.clj

```diff
@@ -5,7 +5,39 @@
     [cmr.common.util :as u :refer [are3]]
     [cmr.ingest.services.jobs :as jobs]))
 
-(deftest create-query-params 
+(deftest create-query-params
   (is (= {"polygon" "-78,-18,-77,-22,-73,-16,-74,-13,-78,-18"
-          "concept-id" "G123-PROV1"} 
+          "concept-id" "G123-PROV1"}
          (#'jobs/create-query-params "polygon=-78,-18,-77,-22,-73,-16,-74,-13,-78,-18&concept-id=G123-PROV1"))))
+
+(deftest email-granule-url-list-test
+ "This tests the utility function that unpacts a list of urls and turns it into markdown"
+ (let [actual (jobs/email-granule-url-list '("https://cmr.link/g1"
+        "https://cmr.link/g2"
+        "https://cmr.link/g3"))
+       expected (str "* [https://cmr.link/g1](https://cmr.link/g1)\n"
+        "* [https://cmr.link/g2](https://cmr.link/g2)\n"
+        "* [https://cmr.link/g3](https://cmr.link/g3)")]
+  (is (= actual expected))))
+
+(deftest create-email-test
+ "This tests the HTML output of the email generation"
+ (let [actual (jobs/create-email-content
+   "cmr-support@earthdata.nasa.gov"
+   "someone@gmail.com"
+   '("https://cmr.link/g1" "https://cmr.link/g2" "https://cmr.link/g3")
+   {:metadata {:SubscriberId "tcherry"}})]
+  (are [exp act] (= exp act)
```

Comment:
not a fan of this syntax. it is more readable to do individual `is` statement when the checking of the clauses don't have anything in common. And you also had the expected and actual switched in the data section.